src/Scriban.Tests/TestParser.cs

```diff
@@ -656,7 +656,29 @@ public void TopLevelAnonymousFunctionCanAccessGloba()
             var template = Template.Parse(templateString);
             Assert.AreEqual("99", template.Render());
         }
+        [Test]
+        public void TestMultiLinePipe()
```

Comment:
Actually, I would prefer these tests to be moved `310-func-pipe.txt` (you don't need to follow the formatting of the tests above).

The reason is that these text files are integrating other tests - e.g verifying that roundtrip is working correctly, or that the async version as well...etc.

Reply:
no problem - will do so tomorrow.

Reply:
Sorry - having a bit of a brain-fade (it's been a long day....)

What is the significance of the "divider" `===` in the input files?  Eg.... 
```
2 | 
===
{{
2 |
}}
```

it is testing both explicit and implicit script mode ?

Reply:
nvm - was overthinking it